pages/index.js

```diff
@@ -19,27 +20,45 @@ export async function getServerSideProps() {
   const URL =
     'https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/ids/mapping.json';
 
+  const membersArr = [];
+  let memberDetailsUrl = '';
+  let resp1 = '';
```

Comment:
Too generic of a name, can we describe more meaningfully what the purpose of this response is?